server/chat-plugins/room-events.ts

```diff
@@ -120,6 +169,7 @@ export const commands: ChatCommands = {
 			}
 			if (!target) return this.errorReply("Usage: /roomevents remove [event name]");
 			target = toID(target);
+			if (getAllAliases(room).includes(target)) return this.errorReply("Use /roomevents removealias to delete aliases.");
```

Comment:
```suggestion
			if (getAllAliases(room).includes(target)) return this.errorReply("To delete aliases, use /roomevents removealias.");
```
JUST in case someone actually thinks you need to type `/roomevents removealias to delete aliases`. This is somewhat of a nitpick though